backend/globaleaks/handlers/public.py

```diff
@@ -360,37 +327,35 @@ def _get_public_resources(store, language):
 
 
 class AhmiaDescriptionHandler(BaseHandler):
+    def _empt_gen(self):
```

Comment:
@NSkelsey when not needed let's use explicit identifier or common abbreviations.

e.g. empt in place of empty is a not needed optimization (save only one char) that simply make the code less naturally readable.